src/components/button/index.tsx

```diff
@@ -72,7 +72,7 @@ class Button extends PureComponent<Props, ButtonState> {
   }
 
   get isFilled() {
-    return !this.isOutline && !this.isLink;
```

Comment:
Check if isOutline method is still used anywhere else...

Reply:
Please check if condition in line 119 (label color) should not be change to !isFilled as well